src/index.ts

```diff
@@ -154,6 +179,22 @@ export const createPersistedQueryLink = (
           includeExtensions: supportsPersistedQueries,
         },
       });
+
+      // If requested, set method to GET if there are no mutations. Remember the
+      // original fetchOptions so we can restore them if we fall back to a
+      // non-hashed request.
+      if (
+        useGETForHashedQueries &&
+        supportsPersistedQueries &&
+        operationIsQuery(operation)
+      ) {
+        operation.setContext(({ fetchOptions = {} }) => {
+          originalFetchOptions = fetchOptions;
+          return { fetchOptions: { ...fetchOptions, method: 'GET' } };
```

Comment:
lets either use Object.assign or the spread notation. I'm more partial to Object.assign because of the unneeded polyfill from typescript

Reply:
Works for me.

Reply:
(Though maybe we can tell TS to not put in the polyfill?)